website/pages/docs/cheatsheet.mdx

```diff
@@ -0,0 +1,124 @@
+import { Callout } from "nextra-theme-docs";
+
+# Cheat sheet
+
+Get started with Scarb by following a cheatsheet of the most important commands.
+
+## Creating new project
+
+```shell copy
+scarb new project/directory
+```
+
+<Callout>
+  There is `scarb init{:shell}` which runs in current directory instead of creating new
+  one.
+</Callout>
+
+## Compiling
+
+```shell copy
+scarb build
+```
+
+Built artifacts will be written to `target/release` directory.
+
+<Callout>`scarb clean{:shell}` cleans `target` directory.</Callout>
+
+### Building CASM
```

Comment:
It's not clear what is the default behavior and outputs

Reply:
done